src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,75 @@
+                  | 2024-05-15     | -04-09    | 2024-04-09 | 0                      |
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>"
+             Then it should respond with 400
+              And response body
+                  """
+                  {
+                      "detail": "Data validation failed. See 'validationErrors' for details.",
+                      "type": "urn:ed-fi:api:bad-request:data",
+                      "title": "Data Validation Failed",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": {
+                          "$.beginDate": [
+                          "The value '<beginDate>' is not valid for BeginDate."
+                          ]
+                      }
+                  }
+                  """
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by date out of specified dates
+            Given there is an unspecified Begin date
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | string         | 2027-04-09 | 2027-04-09 | 0                      |
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=1970-04-09"
+             Then it should respond with 200
+              And response body is an empty array
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by begin date and end date
+            Given there is an invalid Begin date
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | string         | 2024-04-09 | 2024-04-09 | 0                      |
+             When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>&endDate=<endDate>"
+             Then it should respond with 200
+              And response body is an empty array
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by string identifier
+            Given there is a valid Begin date
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | string         | 2024-04-09 | 2024-04-09 | 0                      |
+             When sending a GET request to "/ed-fi/academicWeeks?weekIdentifier=<weekIdentifier>"
+             Then it should respond with 200
+              And response body contains ""weekIdentifier": "string""
+
+
+        @ignore
+        Scenario: Ensure clients can GET information when querying by integer identifier
+            Given there is an invalid Begin date
+                  | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+                  | Week test 1    | 2024-05-13 | 2024-05-13 | 1                      |
+             When sending a GET request to "/ed-fi/academicWeeks?weekIdentifier=<totalInstructionalDays>"
```

Comment:
Use the actual value here: `?totalInstructionalDays=1`.

The Scenario says "querying by integer identifier", but that is not accurate. This is not querying by an identifier.

Line 75 should have a count 1. Suggest using single apostrophe around the string instead of quotation marks, because it is hard to work with `""`. That is, change to `'"totalInstructionalDays": 1'` (also remove the comma, because it could be the last thing in the JSON response).

Reply:
Complete